The list of files with translatable strings in po/Makefile is sorted
and a bashism in has been fixed.
This patch has been applied upstream.

Provided by: Sven Joachim <svenjoac@gmx.de>

---
 po/Makefile |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/po/Makefile
+++ b/po/Makefile
@@ -22,7 +22,7 @@
 	done
 
 $(POFILES_SRC):
-	find ../lib -type f -name \*.sh > $(POFILES_SRC)
+	find ../lib -type f -name \*.sh | sort > $(POFILES_SRC)
 	echo ../backup-manager >> $(POFILES_SRC)
 
 $(PACKAGE).pot: $(POFILES_SRC)
@@ -43,7 +43,7 @@
 .po.mo: backup-manager.pot
 	@echo "Merging $(PACKAGE).pot and $*.po"
 	@msgmerge $*.po $(PACKAGE).pot -o $*.po.new
-	@if [[ "`diff $*.po $*.po.new | grep '[<>]' | wc -l`" -ne 2 ]]; then \
+	@if [ "`diff $*.po $*.po.new | grep '[<>]' | wc -l`" -ne 2 ]; then \
         mv -f $*.po.new $*.po; \
     else \
         rm -f $*.po.new; \
